File: code-coaches/medium/even-numbers/EvenNumbers.cs
using System;
using System.Collections.Generic;

public class EvenNumbers
{
    static string GetEvensStr(string numsStr)
    {
        string[] numsList = numsStr.Split(' ');
        List<string> evenNumsList = new List<string>();

        foreach (var numStr in numsList)
        {
            int num = int.Parse(numStr);
            if (num % 2 == 0)
            {
                evenNumsList.Add(numStr);
            }
        }

        string evenNumsStr = string.Join(" ", evenNumsList);
        return evenNumsStr;
    }

    static void Main(string[] args)
    {
        string numsStr = Console.ReadLine();
        Console.WriteLine(GetEvensStr(numsStr));
    }
}

File: code-coaches/medium/even-numbers/EvenNumbers.swift
func getEvensStr(_ numsStr: String) -> String {
    let numsList = numsStr.split(separator: " ")
    var evenNumsList = [String]()

    for numStr in numsList {
        let num = Int(numStr)!
        if num % 2 == 0 {
            evenNumsList.append(String(num))
        }
    }

    let evenNumsStr = evenNumsList.joined(separator: " ")
    return evenNumsStr
}

func main() {
    let numsStr = readLine()!
    print(getEvensStr(numsStr))
}

main()